src/main/webapp/app/overview/course-exams/course-exam-detail/course-exam-detail.component.ts

```diff
@@ -28,6 +28,6 @@ export class CourseExamDetailComponent {
      * calculate the duration in minutes between the start and end date of the exam
```

Comment:
I think the comment is outdated 
```suggestion
     * calculate the duration in seconds between the start and end date of the exam
```